proxymode/resolver.go

```diff
@@ -16,25 +17,57 @@ type proxyResolver struct {
 	destinationURL url.URL
 }
 
+var acceptedHeaders = map[string]string{
+	"Content-Length": "",
+	"Content-Type":   "",
+	"Last-Modified":  "",
+}
+
+func (pr proxyResolver) pipeRequestProxy(w http.ResponseWriter, remotePath string) error {
+	remoteURL := pr.destinationURL.ResolveReference(&url.URL{Path: remotePath})
+
+	resp, err := http.Get(remoteURL.String())
+	if err != nil {
+		return err
```

Comment:
If we return here without setting a status code we may be returning a 200, but this should probably be a 500.

In general in this method we should be probably producing response errors instead of returning them. The callers neither return errors.

Reply:
Added a response with `http.StatusInternalServerError` status.